notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>It would be great to get interavtive visualisations, using holoviz, or similar, as some of the overlapping data make these graphs difficult to read. Obviously, being able to edit the axes and download the figure is an improvement from the printed version in the paper anyway</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>It would be nice and it could be done in the future. Thanks for your comment.</p>